docs/report/report.tex

```diff
@@ -45,6 +45,105 @@ \subsubsection{Stretch Goals}
+Dabei könnte dann der neue Account zu dem bereits bestehendem Account hinzugefügt und entsprechend in der Datenbank hinterlegt werden.
+
+\subsubsection{Musik Artwork laden}
+Sollte bei der Verwendung von Spotify kein Problem sein, da zu jeder Anfrage von Titeln oder Liedern auch eine Liste von Bildern enthalten ist.\footnote{https://stackoverflow.com/questions/10123804/retrieve-cover-artwork-using-spotify-api}
+
+\subsubsection{Eigene Bilder hochladen}
+Eigene Bilder hochzuladen sollte möglich sein. In unserem Kontext mit Vue.js und Node.js würde das Plugin \textit{vue-picture-input} helfen.
+Mit einem Axios Post könnte das Bild an das Backend gesendet werden. \footnote{https://www.digitalocean.com/community/tutorials/vuejs-uploading-vue-picture-input}
+
+\subsubsection{Spotify Connect Lautsprecher auswählen}
+Das Auswählen von einem spezifischen Spotify Connect Lautsprecher ist möglich.
+Über einen API Call an die Spotify API mit dem Endpunkt \textit{/v1""/me""/player""/device} wird eine Liste von allen verbundenen Geräten geliefert. Über den Endpunkt kann ein entsprechendes Lied zum Abspielen über den jeweiligen \textit{Spotify Connected Speaker} übergeben werden.
+Sollte nicht explizit ein Lautsprecher angegeben werden, so wird der zuletzt aktive genutzt. Dieser hat bei \textit{is\_active} den Wert \textit{true}. \footnote{https://developer.spotify.com/documentation/web-api/guides/using-connect-web-api/}
+
+\subsubsection{Spotify Connect Lautsprecher speichern}
+Die Liste von verbundenen Geräten, die über einen Call an die Spotify API erhalten wird, enthält auch ein eindeutiges Feld \textit{id}, welches sich zusammen mit einem Namen speichern lässt.
+
+\subsubsection{In Cloud Anwendung die jeweilige Box auswählen}
+Bei der Ersteinrichtung der Box könnte sich die Cloud Anwendung mit der MAC Adresse oder einem vom Nutzer gesetzten Namen anmelden.
+Der Box wäre dann die jeweilige Box bekannt und könnte über zum Beispiel den vom Nutzer gewählten Namen wiedergefunden und ausgewählt werden.
+
+\subsubsection{Boxdaten über Cloud Anwendung ändern}
+Um die auf der Box gespeicherten Daten aus der Cloud Anwendung heraus zu ändern, könnte eine API genutzt werden.
+
+\subsubsection{Unterstützung von Youtube Music}
+Eine Umsetzung könnte sich als umständlich erweisen, da es bisher noch keine dedizierte Youtube Music API gibt.
+
+\subsubsection{Unterstützung von Youtube}
+Youtube bietet die Möglichkeit, nach Videos zu suchen \footnote{https://developers.google.com/youtube/v3/}. Um diese Videos auf dem Raspberry als Musik abzuspielen würde sich \textit{youtube-dl} zum downloaden der Videos als \textit{.mp3} Dateien und \textit{omxplayer} zum Abspielen anbieten.
+Hierfür wäre allerdings ein entsprechender Lautsprecher am Raspberry erforderlich. Ein vergleichbares System zu Spotify Connect existiert derzeit in dieser Form noch nicht für Youtube.
```

Comment:
```suggestion
Hierfür wäre allerdings ein entsprechender Lautsprecher am Raspberry erforderlich. Ein vergleichbares System zu Spotify Connect existiert derzeit noch nicht.
```

Reply:
resolved in commit e28b6a9d9f4295a56402e2508b5f8250440df0cc